TL improvement (#537)
07th-mod/higurashi-console-arcs: Update/omot_008_choice2.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500460", 256, TRUE);
	OutputLine(NULL, "「大局的に見れば…なんて、勝手な言い分だと思います。",

> 			NULL, "\"I know it would be selfish of me... to make a excuse like \"looking it from the big picture.\"", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I know it would be selfish of me... to make an excuse like \"looking at it from the big picture.\"", Line_WaitForInput);
```

			NULL, "\"There may be another good way...", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100622", 256, TRUE);
	OutputLine(NULL, "それに、俺は別に村の連中にそんなやり方で信頼されたいなんて思わない」",

> 			NULL, " Plus, I don't want to win the trust of the villagers in such a way.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Plus, I don't want that to be the way I win the trust of the villagers.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100625", 256, TRUE);
	OutputLine(NULL, "「敵がどういうやつかはまだ分からないけど、俺たちの幸せをぶち壊そうとするやつは絶対に許さないし、絶対に負けたりしない！」",

> 			NULL, "\"I still don't know who this enemy is, but I'll never forgive them for trying to destroy our happiness, and I'll never be a loser!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"I still don't know who this enemy is, but I'll never forgive them for trying to destroy our happiness, and I'll never admit defeat!\"", Line_Normal);
```
